### PR TITLE
Remove automatic lint baseline creation

### PR DESCRIPTION
**Pull request by f2rf2r**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Change Log
 ==========
+
+Version 1.2.2 *(2019-1-11)*
+----------------------------
+* Changed: Don't automatically create lint baseline file. Projects can manually specify a baseline file in `lintOptions` if they want it.
```
Comment (mpeng3) [inline CHANGELOG.md]: It might be nice to include more explicit instructions about how to do this in the README.

**Review by mpeng3 — COMMENTED** [review]
Can you remind me what the reason behind this change was?  Was it because we don't want to blast the baseline file that people may have already created each time this is run, or was it more because we'd prefer that projects don't have a baseline at all so this (by default) doesn't hide any of their lint warnings?

**Comment by David-Zou-Intrepid** [comment]
This is so that projects that don't want a baseline file aren't forced to use one

**Comment by f2rf2r** [comment]
This is so that projects that don't want a baseline file aren't forced to use one

File: CHANGELOG.md (hunk 1 above)
Comment (f2rf2r) [reply CHANGELOG.md]: I added an example in the changelog

**Review by mpeng3 — APPROVED** [review]
:shipit:

**Pull request merged**